content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md

```diff
@@ -0,0 +1,1493 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Node"
+content_type: "api_reference"
+description: "Node 是 Kubernetes 中的工作节点。"
+title: "Node"
+weight: 1
+---
+
+<!-- 
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Node"
+content_type: "api_reference"
+description: "Node is a worker node in Kubernetes."
+title: "Node"
+weight: 1
```

Comment:
Missed `auto_generated: true` in original metadata.

Reply:
The sentence is meaningless and was suggested to be removed by other reviewers